webpack.config.js

```diff
@@ -107,17 +112,35 @@ var config = (module.exports = {
       // NOTE @kdoh - 7/24/18
       // icepick 2.x is es6 by defalt, to maintain backwards compatability
       // with ie11 point to the minified version
-      icepick: __dirname + "/node_modules/icepick/icepick.min"
+      icepick: __dirname + "/node_modules/icepick/icepick.min",
     },
   },
 
+  // optimization: {
```

Comment:
Can we just remove this or will we want to try and do something with this later?

Reply:
That reminds me I need to make sure the bundle size is approximately the same.